account_banking_uk_hsbc/mt940_parser.py

```diff
@@ -50,18 +50,18 @@ def __init__(self):
                            r"(?P<startingbalance>[\d,]{1,15})")
 
         # Transaction
-        recparse["61"] = r"""\
-:(?P<recordid>61):\
-(?P<valuedate>\d{6})(?P<bookingdate>\d{4})?\
-(?P<creditmarker>R?[CD])\
-(?P<currency>[A-Z])?\
-(?P<amount>[\d,]{1,15})\
-(?P<bookingcode>[A-Z][A-Z0-9]{3})\
-(?P<custrefno>[%(ebcdic)s]{1,16})\
-(?://)\
-(?P<bankref>[%(ebcdic)s]{1,16})?\
-(?:\n(?P<furtherinfo>[%(ebcdic)s]))?\
-""" % (patterns)
+        recparse["61"] = (r"""
+:(?P<recordid>61):
+(?P<valuedate>\d{6})(?P<bookingdate>\d{4})?
+(?P<creditmarker>R?[CD])
+(?P<currency>[A-Z])?
+(?P<amount>[\d,]{1,15})
+(?P<bookingcode>[A-Z][A-Z0-9]{3})
+(?P<custrefno>[%(ebcdic)s]{1,16})
+(?://)
+(?P<bankref>[%(ebcdic)s]{1,16})?
+(?:\n(?P<furtherinfo>[%(ebcdic)s]))?
+""" % (patterns)).replace('\n','')
```

Comment:
Interesting bug. The fix looks good to me.

Can you add a space after the comma for pep8 compliance?